include/swift/ClangImporter/ClangImporter.h

```diff
@@ -242,8 +242,7 @@ class ClangImporter final : public ClangModuleLoader {
   ///
   /// \sa clang::GeneratePCHAction
   bool emitBridgingPCH(StringRef headerPath,
-                       StringRef outputPCHPath,
-                       clang::DiagnosticConsumer *Diags = nullptr);
```

Comment:
David added the extra parameter for passing a new DiagnosticConsumer. This was to workaround an issue where at the point where we were calling emitBridgingPCH(), the clang instance of the importer hadn't initialized the clang ASTContext yet, which triggered a crash in `ClangDiagnosticConsumer.cpp` for initializing the ClangDiagRenderer.
However, this prevented reusing the normal consumer for the clang diagnostics when creating the PCH, so it was not desirable (they were not getting serialized for example).
I noticed that the ClangDiagRenderer did not actually need the ASTContext, it only needed some options, see changes in `ClangDiagnosticConsumer.cpp` in this PR, so we didn't need to pass a different DiagnosticConsumer any more.